src/Microsoft.Extensions.CommandLineUtils/CommandLine/CommandLineApplication.cs

```diff
@@ -52,14 +65,18 @@ public CommandLineApplication(bool throwOnUnexpectedArg = true)
             return command;
         }
 
-        public CommandOption Option(string template, string description, CommandOptionType optionType)
+        public CommandOption Option(string template, string description, CommandOptionType optionType, bool inherited = false)
```

Comment:
This is a breaking change (API/binary compatibility). Consider adding an overload